notebooks/samples/301 - CIFAR10 CNTK CNN Evaluation.ipynb

```diff
@@ -183,7 +174,7 @@
    "execution_count": null,
    "metadata": {
     "mml-deploy": "hdinsight",
-    "collapsed": false
+    "collapsed": true
```

Comment:
why are these collapsed now?

Reply:
I made the verification thing enforce a rule that `mml-deploy` with a `local` has `false` and with `hdinsight` it should have `true`.  The idea is to see just the local version which is more presentable since it doesn't have the hdinsight hacks.  I'm not sure about it though, so that's a question for @drdarshan whether there is a technical reason to change this.